templates/web/base/report/form/user_loggedout_by_email.html

```diff
@@ -47,13 +45,13 @@
         <p class='form-error'>[% field_errors.username_register %]</p>
     [% END %]
 
-    <label for="form_email">[% loc('Your email') %]<span class="hidden-js" id="js-optional-email"> [% loc('(optional)') %]</span></label>
```

Comment:
I think these were hidden to begin with so that people wouldn't potentially get confused seeing both listed as optional before they picked an update method above. You can leave the class on here, and instead of hide/show in the JS, do addClass/removeClass/toggleClass hidden-js, I think?